.azure-pipelines/publish.yml

```diff
@@ -0,0 +1,53 @@
+resources:
+  repositories:
+  - repository: 1esPipelines
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+  parameters:
+    pool:
+      name: DevDivPlaywrightAzurePipelinesUbuntu2204
+      os: linux
+    sdl:
+      sourceAnalysisPool:
+        name: DevDivPlaywrightAzurePipelinesWindows2022
+        # The image must be windows-based due to restrictions of the SDL tools. See: https://aka.ms/AAo6v8e
+        # In the case of a windows build, this can be the same as the above pool image.
+        os: windows
+    stages:
+    - stage: Stage
+      jobs:
+      - job: HostJob
+        templateContext:
+          outputs:
+          - output: buildArtifacts
+            # The exact name "CodeAnalysisLogs" is required for the Sarif Results Viewer Extension for Azure Pipelines
+            # to find the .sarif files our accessibility test produces.
+            displayName: publish sarif results
+            # "test-results" is not a required convention; it just happens to be where our tests write .sarif files to.
+            PathtoPublish: $(Build.SourcesDirectory)/test-results
+            ArtifactName: CodeAnalysisLogs
+            condition: succeededOrFailed()
+        steps:
+        - task: UseNode@1
+          inputs:
+            version: '18.x'
+          displayName: 'Install Node.js'
+        - script: npm ci
+          displayName: Install
+        - script: npm run build
+          displayName: Build
+        - script: npx playwright install --with-deps
+          displayName: 'Install Playwright Browsers'
+        - script: npx playwright test
```

Comment:
Perhaps only run accessibility tests here? I am not sure how the integration works with the insights tool.